### ISSUE TITLE
Add option to display file even if bat thinks it is binary

### ISSUE THREAD

**Issue by NightMachinary** [sharkdp/bat#623]
I have a file with this content:
```sh
# cat -v sth
^@this is 1.
this is 2.
this is 3.
^@hi
^@blue boy
```

And `bat -A sth` returns:
![image](https://user-images.githubusercontent.com/36224762/62610317-b34c8180-b918-11e9-9f2b-bc0e0a38fa2b.png)

**Comment by sharkdp** [comment]
We do this on purpose to prevent users from accidentally "printing" binary files to the terminal that could mess up their terminal emulator.

But adding an option to overwrite this behavior sounds very reasonable. Also, I agree that it would be great if `bat -A` would show the file even if it is binary.

**Comment by sharkdp** [comment]
see also #619

**Comment by sharkdp** [comment]
Let's start by changing `bat -A` such that binary files will be displayed. Non-UTF-8 byte sequences would be replaced by a hex representation (e.g.`\xFE\xEF`).

**Comment by NightMachinary** [comment]
👍🏽🤞🏻

**Comment by sharkdp** [comment]
This can now be done with `bat -A`, see #640 for details.

**Comment by sharkdp** [comment]
Released in [bat v0.12](https://github.com/sharkdp/bat/releases/tag/v0.12.0).
